webpack.config.js

```diff
@@ -22,20 +22,21 @@ module.exports = {
         'css?sourceMap!less?sourceMap'
       ), exclude: ['node_modules'] }
     ],
-    preLoaders: [
-      { test: /\.jsx$/, loaders: ['eslint-loader'], exclude: ['node_modules'] }
+    rules: [
+      { test: /\.jsx$/, enforce: 'pre', loader: 'eslint-loader', exclude: ['node_modules'] },
+      { test: /\.less$/, use: ExtractTextPlugin.extract({
```

Comment:
new way to transform less, apparently?

Reply:
There is a lot going on here, and I have some questions.

1. What's the difference between rules vs preLoaders, and what does preLoaders mean when compared to the loaders above.
2. What's the relationship between rules and loaders above.
3. We have a less loader in loaders above, does something like rules remove that?
4. I wonder if loaders is completely replaced by rules, and we can just remove it with what you've added?

Reply:
Yeah, in the docs, I see "rules should be used instead of loaders" and use `enforce: 'pre'` for preLoaders like you have.

The only thing I think  you need to do is remove the `loaders` above, and move the json loader from the old `loaders` into the new `rules`

Reply:
Cool, if that worked as expected, I see no reason why not to send it to staging.